README.md

```diff
@@ -226,6 +226,41 @@ Follow Helm Chart installation instructions [here](https://docs.openshift.com/co
 
 ## Additional information
 
+### Additional Workflow Namespaces
+
+If you want to deploy workflows on other namespaces, make sure you follow these steps:
+
+ - For the cluster Sonataflow services to accept the traffic from the workflows, set this label:
```

Comment:
Can you please reword a bit this sentence? It's difficult to read.
Maybe something like "To allow cluster Sonataflow services to accept traffic from workflows, set this label in the desired workflow:"